Gemfile

```diff
@@ -55,6 +55,7 @@ end
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/204615114-504d5e4b-f4bf-4a53-b0b1-8e7cb616aa30.png)

- [ ] Your readme file is not descriptive enough :fearful: Kindly add the missing sections in your readme file using this [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) :+1: